package.json

```diff
@@ -22,10 +22,6 @@
   ],
   "author": "joshwnj",
   "license": "MIT",
-  "peerDependencies": {
```

Comment:
Forgot I had removed that. It doesn't really need to be removed, but those packages are a devDependency if the example is to be built. peerDependencies was changed in NPM 3, doesn't install the packaged just issues warnings. The peer part can remain as a duplicate